.changeset/sweet-monkeys-reply.md

```diff
@@ -0,0 +1,7 @@
+---
+'@makeswift/runtime': patch
+---
+
+Button component and Link control now hydrate page links with the locale, if present. Brings automatic link localization to App Router, while still supporting Pages Router.
+
+Also resolves issue where rewritten host API requests are unauthorized due to not checking the request header for the secret.
```

Comment:
Not a big deal but we should try and include changesets in the commit that introduce the change so that when they show up in the change log the commit shows the appropriate diff.

Reply:
Ah thanks, I thought changesets were set on a PR granularity, not per commit. I can adjust this.